resources/builder.inc.sh

```diff
@@ -40,6 +40,12 @@ function _builder_init() {
   else
     builder_use_color false
   fi
+
+  # Set shared Meson package cache (works with Meson >= 1.3)
+  if [[ -z ${MESON_PACKAGE_CACHE_DIR:-} ]]; then
+    export MESON_PACKAGE_CACHE_DIR="${XDG_CACHE_HOME:-${HOME}/.cache}/keyman/builder"
```

Comment:
Is `~/.cache` a safe path? We have already a ~/.keyman path used for nvm (see https://github.com/keymanapp/keyman/blob/aa506722ca5210f0ee7e50f52147da9b7f499a5a/docs/build/node.md#L33) so maybe it'd be better to share that?